client/src/App.js

```diff
@@ -20,6 +21,7 @@ function App() {
             <PrivateRoute path="/setting" component={SettingPage} />
             <Route path="/signup" component={Signup} />
             <Route path="/login" component={Login} />
+            <Route path="/testingmaterial" component={SortToggle} />
```

Comment:
I don't think this is necessary to leave in for merge?

Reply:
Oops my bad, it will DEFINITELY be removed